agot-bg-game-server/src/client/game-state-panel/ResolveSingleMarchOrderComponent.tsx

```diff
@@ -99,26 +99,11 @@ export default class ResolveSingleMarchOrderComponent extends Component<GameStat
 
         return (
             <Col xs={12} className="text-center">
-                <OverlayTrigger overlay={
-                    <Tooltip id={"leave-power-token"}>
-                        {reason == "already-capital" ? (
-                            <>Your capital is always controlled by your house, thus not requiring a Power
-                                token to be left when leaving the area to keep control of it.</>
-                        ) : reason == "already-power-token" ? (
-                            <>A Power token is already present.</>
-                        ) : reason == "no-power-token-available" ? (
-                            "You don't have any available Power token."
-                        ) : reason == "not-a-land" ? (
-                            "Power tokens can only be left on land areas."
-                        ) : reason == "no-all-units-go" ? (
-                            "All units must leave the area in order to leave a Power token."
-                        ) : "Leaving a Power token in an area maintain the control your house has on it, even"
-                            + " if all units your units leave the area."}
-                    </Tooltip>
-                }>
+                <OverlayTrigger overlay={this.renderTooltip(reason)}>
                     <Form.Check
+                        id="chk-leave-pt"
                         label="Leave a Power Token"
-                        checked={this.leavePowerToken}
+                        checked={success && this.leavePowerToken}
```

Comment:
This is not needed, I think.

Reply:
Well.. As described on Discord you may run in the situation that the checkbox is ticked because the value for this.leavePowerToken is persisted over a whole action phase. When you perform a second march and the checkbox gets disabled there the user might be confused as he thinks he leaves a power token even if it's not needed or just not possible and he can't deactivate it! See #89

Reply:
Indeed I didn't tought of that!

I would then set `this.leavePowerToken` to `false` when `selectedMarchOrderRegion` is set to `null`.

Reply:
Ok.. will do so at about 9 this evening